api/app/signals/apps/signals/models/__init__.py

```diff
@@ -31,6 +32,9 @@
 
 # Satisfy Flake8 (otherwise complaints about unused imports):
 __all__ = [
+    'Area',
+    'AreaHistory',
```

Comment:
The AreaHistory model is not in this PR anymore, so it can be removed from the list